docs/_developers/dns-discovery-setup.md

```diff
@@ -0,0 +1,113 @@
+---
+title: DNS Discovery Setup Guide
+sort_key: C
+---
+
+This document explains how to set up an [EIP 1459][dns-eip] node list using the cmd/devp2p
```

Comment:
Can you link to the tool? Do you mean https://github.com/ethereum/go-ethereum/tree/master/cmd/p2psim? Or is it on another branch?

Reply:
The link to the EIP is right there. I will add more info to this document once we have a way to actually use DNS discovery in geth.